### DISCUSSION TITLE
阿里云盘Open的failed link: failed get link: TooManyRequests

### DISCUSSION THREAD

**Discussion by Kotori0629** [AlistGo/alist#4059] [Q&A]
这个情况是由啥引起？是否有什么解决方式？

**Comment by Arminosi** [comment]
同样出现了这个问题 可能最近阿里云API做了限制措施

**Reply by Kotori0629** [reply]
em 我关帖是因为看到了官网对这个的说明，不过前段时间没问题 突然今天出现的

**Reply by Bssn520** [reply]
> em 我关帖是因为看到了官网对这个的说明，不过前段时间没问题 突然今天出现的

请问解决了吗？我也遇到了这个问题，可以交流交流

**Reply by Kotori0629** [reply]
> > em 我关帖是因为看到了官网对这个的说明，不过前段时间没问题 突然今天出现的
> 
> 请问解决了吗？我也遇到了这个问题，可以交流交流

未解决，返回再点击文件 多试几次又能好了

**Reply by Arminosi** [reply]
没有 我也是今天出现了这个问题所以来github看看 官网注明的是保存/编辑会计数请求 但是查看文件不算
我今天挂载的阿里云存储就只是点进文件就出现请求过多 可能阿里云那边今天突然改了什么

**Reply by Bssn520** [reply]
害，目前也没找到什么办法能解决🥲

**Reply by Arminosi** [reply]
急用暂时先转123存储吧

**Reply by Kotori0629** [reply]
> 急用暂时先转123存储吧

我左转OneDrive了

**Reply by Bssn520** [reply]
可是耐不住阿里生态好

> > 急用暂时先转123存储吧
> 
> 我左转OneDrive了

**Reply by Kotori0629** [reply]
> 可是耐不住阿里生态好
> 
> > > 急用暂时先转123存储吧
> > 
> > 
> > 我左转OneDrive了

观望观望

**Reply by mondayice** [reply]
我也出现了。

**Comment by chuanyilang** [comment]
问题解决了吗，刚遇到

**Comment by SfYangYang** [comment]
我也遇到了，签到领了一天的SVIP影音播放体验，结果解决了。难道以后只有买会员才能用api播放了吗？

**Comment by 123385918** [comment]
还真是  签到领SVIP影音播放  就解决了  感觉以后只有会员能用了啊

**Reply by doge-8** [reply]
领了能看部分，但是还有很多片放不出来

**Comment by sunpm** [comment]
年费会员也不行。。。。。

**Comment by doge-8** [comment]
我才刚学会这好方法就G了，干啊

**Comment by Arminosi** [comment]
有可能阿里云的人也在关注这个帖子哈哈哈

**Comment by wanglei-github** [comment]
我也出现了，昨晚刚出现，基本没法通过kodi看，但我换了其他的，就没有出现 tooManyRequests的问题。
老板们，帮忙看看呗，alist觉得挺好用的，老板们，加油啊

**Comment by yangbvv** [comment]
昨晚不好用，今天又恢复了

**Reply by doge-8** [reply]
好像比昨天好点，但是快进不行。重新打开能放了

**Comment by Kotori0629** [comment]
测试了下 又可以了，现在就是和官网写的一样的 10次限制

**Comment by Malcolmwongzz** [comment]
我也出现了，求教怎么解决，目前只能不停刷新，偶尔能成功

**Comment by oneisall8955** [comment]
所以到底怎么解决，缓存 download_url 可勉强解决吧？
